homeassistant/components/lastfm/sensor.py

```diff
@@ -60,11 +89,24 @@ class LastFmSensor(SensorEntity):
     _attr_attribution = "Data provided by Last.fm"
     _attr_icon = "mdi:radio-fm"
 
-    def __init__(self, user: User, lastfm_api: LastFMNetwork) -> None:
+    def __init__(self, user: User, entry_id: str) -> None:
         """Initialize the sensor."""
-        self._attr_unique_id = hashlib.sha256(user.name.encode("utf-8")).hexdigest()
-        self._attr_name = user.name
         self._user = user
+        self._attr_unique_id = hashlib.sha256(user.name.encode("utf-8")).hexdigest()
+        self._attr_name = f"lastfm_{user.name}"
```

Comment:
```suggestion
        self._attr_name = user.name
```
Does not look like a very friendly name.
Don't change the name in this PR